src/config/dynamic-theme-fixes.config

```diff
@@ -16021,6 +16031,13 @@ INVERT
 
 ================================
 
+tasks.google.com
+
+INVERT
+.tWyAef
```

Comment:
Please try to avoid such selectors, they have a low-guarantee that they will still work over a few months.

Reply:
I don't think I can figure out a way of doing so with this and the other one on my own, so I think I'll erase them from here and submit an issue instead. If it's of any help, I've had these written down since around December from last year, https://github.com/darkreader/darkreader/pull/7678, and they still work.